sklearn/preprocessing/tests/test_data.py

```diff
@@ -2619,3 +2619,9 @@ def test_power_transformer_constant_feature(standardize):
             assert_allclose(Xt_, np.zeros_like(X))
         else:
             assert_allclose(Xt_, X)
+
```

Comment:
Please also add a test for the warning : https://github.com/scikit-learn/scikit-learn/issues/23319#issuecomment-1464933635